engine/src/cmd/unit_generic.cpp

```diff
@@ -5147,13 +5145,11 @@ void Unit::UpdatePhysics3(const Transformation &trans,
       }
   }
 
-  // TODO: this should never happen
-  if (health.max_health <= 0) {
+  // TODO: lib_damage
+  // I merged two conditions for explode and kill here
```

Comment:
Hmm. I hope that is correct

Reply:
should be more or less, it's just a matter of perspective.